cloudevents/formats/cbor-format.md

```diff
@@ -0,0 +1,199 @@
+which this mapping leans on.
+
+### 2.2. Type System Mapping
+
+The [CloudEvents type system][ce-types] MUST be mapped to CBOR types as follows,
+with exceptions noted below.
+
+| CloudEvents   | CBOR                                                         |
+| ------------- | ------------------------------------------------------------ |
+| Boolean       | [Simple value][cbor-fpnocont] number `20` for `false` and number `21` for `true` |
+| Integer       | [Major type 0][cbor-major-types] for positive integers and [Major type 1][cbor-major-types] for negative integers                         |
+| String        | [Major type 3][cbor-major-types]                             |
+| Binary        | [Major type 2][cbor-major-types]                             |
+| URI           | [Major type 6][cbor-major-types] [tagged][cbor-tagging] as a URI (tag number `32`)|
+| URI-reference | [Major type 3][cbor-major-types] |
+| Timestamp     | [Major type 6][cbor-major-types] [tagged][cbor-tagging] as a [Standard Date/Time String][cbor-standard-datetime] (tag number `0`)|
+
+Unset attributes MAY be encoded to the [Simple value][cbor-fpnocont] value `22` (`null`). When decoding
+attributes and a `null` value is encountered, it MUST be treated as the
+equivalent of unset or omitted.
+
+Extension specifications MAY define secondary mapping rules for the values of
+attributes they define, but MUST also include the previously defined primary
+mapping.
+
+For instance, the attribute value might be a data structure defined in a
+standard outside of CloudEvents, with a formal CBOR mapping, and there might be
+risk of translation errors or information loss when the original format is not
+preserved.
+
+An extension specification that defines a secondary mapping rule for CBOR, and
+any revision of such a specification, MUST also define explicit mapping rules
+for all other event formats that are part of the CloudEvents core at the time of
+the submission or revision.
+
+If required, the CloudEvents type can be determined by inference using the rules
+from the mapping table. The value is compatible with the respective CloudEvents type when the
+mapping rules are fulfilled.
+
+### 2.4. CDDL Schema Validation
+
+The CloudEvents [CDDL][cddl-spec] for the spec is defined under
+[cloudevents.cddl](cloudevents.cddl) and contains the definitions for validating events in
+CBOR.
+
+## 3. Envelope
+
+Each CloudEvents event can be wholly represented as a CBOR data item.
+
+Such a representation MUST use the media type `application/cloudevents+cbor`.
+
+All REQUIRED and all not omitted OPTIONAL attributes in the given event MUST
+become members of the CBOR data item, with the respective CBOR item member name
+matching the attribute name, and the member's type and value being mapped using
+the [type system mapping](#22-type-system-mapping).
+
+OPTIONAL not omitted attributes MAY be represented as a `null` CBOR simple value.
```

Comment:
is this a repeat of line 65?

Reply:
not sure, this is how it is in the JSON spec